socorro/cron/jobs/ftpscraper.py

```diff
@@ -368,7 +370,13 @@ def run(self, date):
             )
 
         if self.config.cachedir:
-            self.config.logger.debug('Cache hits: %d' % self.cache_hits)
+            total = float(self.cache_hits + self.cache_misses)
+            self.config.logger.debug('Cache: hits: %d (%2.2f%%) misses: %d (%2.2f%%)' % (
+                self.cache_hits,
+                self.cache_hits / total * 100,
+                self.cache_misses,
+                self.cache_misses / total * 100,
+            ))
```

Comment:
It's not like you didn't know this but one convenient way to do percentages in python 2.7 of ints is to do:
```
p = 100.0 * one_int / another_int
```